cognitive/src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -457,4 +457,113 @@ class PIISuiteV4 extends TestBase with DataFrameEquality with TextKey {
     .collect()
     assert(tdf.length == 3)
   }
+}
+
+class HealthcareSuiteV4 extends TestBase with DataFrameEquality with TextKey {
+
+  import spark.implicits._
+
+  lazy val df3: DataFrame = Seq(
+    ("en", "20mg of ibuprofen twice a day")
+  ).toDF("lang", "text")
+
+  lazy val df4: DataFrame = Seq(
+    ("en", "1tsp of Tylenol every 4 hours")
+  ).toDF("lang", "text")
+
+  lazy val df5: DataFrame = Seq(
+    ("en", "6-drops of Vitamin B-12 every evening")
+  ).toDF("lang", "text")
+
+  val options: TextAnalyticsRequestOptionsV4 = new TextAnalyticsRequestOptionsV4("", true, false)
```

Comment:
I think i removed this from API

Reply:
@mhamilton723 we removed this from the c'tor but @skonigs has been working on adding the `setOptions()` on LOC 477.